test/conformance/filesystem_test.go

```diff
@@ -0,0 +1,96 @@
+// +build e2e
+
+/*
+Copyright 2018 The Knative Authors
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package conformance
+
+import (
+	"fmt"
+	"testing"
+
+	"github.com/knative/serving/test"
+	"github.com/knative/serving/test/types"
+)
+
+func verifyModeString(resp string, expected string) error {
+	if len(resp) != len(expected) {
+		return fmt.Errorf("length of expected and response string don't match. Expected Response: %q Received Response: %q Expected length: %d Received length: %d",
+			expected, resp, len(expected), len(resp))
+	}
+
+	for index := range expected {
+		if string(expected[index]) != "*" && expected[index] != resp[index] {
+			return fmt.Errorf("mode strings don't match at Index: %d. Expected: %q Received Response: %q",
```

Comment:
```suggestion
			return fmt.Errorf("mode[%d] = %c, want = %c", index, expected[index], resp[index])
```

Reply:
Cleaned this one up and the one above it.